### PR TITLE
Sara/mkt 348 q2 launch pricing and billing

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/semgrep-app/pricing-and-billing.mdx [hunk 1]
```diff
@@ -45,35 +45,47 @@ Members who do **not** run any scans on their code do not need a seat and are no
...
 
 Notes:
 
-* **Named rules** are rules that have been saved through the [Playground](https://semgrep.dev/editor). The Playground is an online tool where you can write your rule patterns and test them on sample code. Named rules created by Community tier members are public and available in the Playground.
-* [The Editor](../editor) is a live, interactive tool for writing rules. The Editor is free for all tiers, but writing and managing **private rules** is a **Team/Enterprise tier** feature.
+* **Custom rules** are rules that have been saved through either the [Playground](https://semgrep.dev/playground) or the Editor. The Playground and the [Editor](../editor) are online tools where you can write your rule patterns and test them on sample code. You must be signed in to use the Editor.
+* Developers can create fixes to their rules through an `autofix` key.For example, when banning outdated libraries, setting an autofix value with the correct library will replace the banned library.
+* Semgrep App provides this functionality with **Autofix through PR/MR comments**. These comments  appear as suggestions that developers can commit with a single click.
```
Comment (adamkvitek) [inline docs/semgrep-app/pricing-and-billing.mdx:75]:
```suggestion
* Semgrep App provides this functionality with **Autofix through PR/MR comments**. These comments appear as suggestions that developers can commit with a single click.
```

**Review by adamkvitek — COMMENTED** [review]
@s-santillan left a few comments for very minor things. Thank you!

File: docs/semgrep-app/pricing-and-billing.mdx [hunk 2]
```diff
@@ -45,35 +45,47 @@ Members who do **not** run any scans on their code do not need a seat and are no
...
+| **Developer feedback** | ❌ no | ✔️ yes | ✔️  yes |
+| **Private rules** | ❌ no | ✔️ yes | ✔️  yes |
 | **Custom language support** | ❌ no | ❌ no | ✔️  yes |
+| **Findings retention** | 1 month | 1 year | 1 year+ (custom) |
 
 Notes:
 
-* **Named rules** are rules that have been saved through the [Playground](https://semgrep.dev/editor). The Playground is an online tool where you can write your rule patterns and test them on sample code. Named rules created by Community tier members are public and available in the Playground.
-* [The Editor](../editor) is a live, interactive tool for writing rules. The Editor is free for all tiers, but writing and managing **private rules** is a **Team/Enterprise tier** feature.
+* **Custom rules** are rules that have been saved through either the [Playground](https://semgrep.dev/playground) or the Editor. The Playground and the [Editor](../editor) are online tools where you can write your rule patterns and test them on sample code. You must be signed in to use the Editor.
```
[the remark this answers is not in the document]
Comment (s-santillan) [reply docs/semgrep-app/pricing-and-billing.mdx]: @adamkvitek nice catch!! thank you :)

**Comment by s-santillan** [comment]
Thanks @adamkvitek! Have committed/resolved everything :)

**Review by adamkvitek — APPROVED** [review]

**Pull request merged**